_posts/2020-03-03-bazel-2-2.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: posts
+title: "Bazel 2.2"
+authors:
+  - philwo
+---
+
+[Bazel 2.2](https://github.com/bazelbuild/bazel/releases/tag/2.2.0) has just
+been released.
+
+Bazel 2.2 is intended to be backward-compatible with Bazel 2.0. There are no
+known breaking changes in this release.
+
+## Incompatible changes
+
+*   The flag `--incompatible_windows_bashless_run_command` has been removed. It
+    was flipped in Bazel 1.0
+*   The flag `--incompatible_windows_native_test_wrapper` has been removed. It
+    was flipped in Bazel 1.0
+*   The flag `--incompatible_disallow_dict_lookup_unhashable_keys` has been
+    removed.
+*   The flag `--incompatible_remap_main_repo` has been removed.
+*   The flag `--incompatible_disallow_unverified_http_downloads` has been
+    removed.
+*   Added the flag `--incompatible_applicable_licenses`. We plan to flip this
+    flag in Bazel 4.0. See
+    [#10687](https://github.com/bazelbuild/bazel/issues/10687) for details.
+*   Added the flag `--incompatible_remove_ram_utilization_factor`. We plan to
+    flip this flag in Bazel 3.0. See
+    [#10730](https://github.com/bazelbuild/bazel/issues/10730) for details.
+*   Added the flag
```

Comment:
nit. don't start sentences with a verb

> The flag `x` is added.

I'd put the GitHub link on the flag name (to avoid "see #number"). Serge recommended to avoid future promisses (https://www.bazel.build/release-notes.html). The information of estimated flag flip time is in the link anyway.